system/views/admin/errorLogs/index.cfm

```diff
@@ -34,6 +35,9 @@
 							</a>
 						</td>
 						<td>
+							#left( log.message, "30" )#...<span class="fa fa-caret-down" data-rel="popover" data-trigger="hover" data-placement="left" data-content="<pre>#log.message#</pre>" data-html="true"></span>
```

Comment:
We need to make sure that we escape the log message when inserting as an attribute in HTML:

```
... data-content="#HtmlEditFormat( '<pre>#log.message#</pre>' )#" ...
```
```